build.zig

```diff
@@ -32,20 +32,21 @@ pub fn build(b: *std.Build) !void {
         });
         module.addImport("mach-sysjs", sysjs_dep.module("mach-sysjs"));
     } else {
-        const mach_glfw_dep = b.dependency("mach_glfw", .{
-            .target = target,
-            .optimize = optimize,
-        });
-        const gamemode_dep = b.dependency("mach_gamemode", .{
-            .target = target,
-            .optimize = optimize,
-        });
+        const mach_glfw_dep = b.dependency("mach_glfw", .{ .target = target, .optimize = optimize });
+        const gamemode_dep = b.dependency("mach_gamemode", .{ .target = target, .optimize = optimize });
+        const x11_headers = b.dependency("x11_headers", .{ .target = target, .optimize = optimize });
+        const wayland_headers = b.dependency("wayland_headers", .{ .target = target, .optimize = optimize });
         module.addImport("mach-glfw", mach_glfw_dep.module("mach-glfw"));
         module.addImport("mach-gamemode", gamemode_dep.module("mach-gamemode"));
+        module.linkLibrary(x11_headers.artifact("x11-headers"));
+        module.linkLibrary(wayland_headers.artifact("wayland-headers"));
+        module.addCSourceFile(.{ .file = .{ .path = "src/platform/wayland/wayland.c" } });
+
+        const test_platform = b.option(App.Platform, "test_platform", "The mach core platform to use when running tests");
 
         const main_tests = b.addTest(.{
             .name = "core-tests",
-            .root_source_file = .{ .path = sdkPath("/src/main.zig") },
+            .root_source_file = .{ .path = "src/main.zig" },
```

Comment:
The removal of it here is correct. In every other location it should remain I think